packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Utility Actions
+intro: Utility actions for a webpage including print, email, RSS and more.
+---
+
+import {
+  DownloadIcon,
+  EmailIcon,
+  LinkIcon,
+  PrintIcon,
+  RssIcon,
+  WhatsNewIcon,
+} from '@cmsgov/design-system';
+
+## Guidance
+
+### Examples
+
+<EmbeddedExample>
+  <div className="ds-l-row">
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
```

Comment:
I meant this should use our Button component with the ghost variable. The component comes with a11y enhancements missing here as well.